src/pyhf/workspace.py

```diff
@@ -743,3 +743,27 @@ def sorted(cls, workspace):
         newspec['observations'].sort(key=lambda e: e['name'])
 
         return cls(newspec)
+
+
+def build(model, data, name='measurement'):
```

Comment:
maybe make this a `@classmethod` so we can return a `cls(....)` ?

Reply:
this returns pure JSON, not a workspace object, should we change it?

Reply:
fix the docstring then? it wasn't clear to me. Although for a python API - it makes sense to return the workspace object. This is dumpable as JSON for printing to the screen if needed, etc.